docs/guides/how_to_guides/creating_batches/how_to_load_a_database_table_view_or_query_result_as_a_batch.rst

```diff
@@ -5,134 +5,246 @@ How to load a database table, view, or query result as a batch
 
 This guide shows how to get a :ref:`batch <reference__core_concepts__batches>` of data that Great Expectations can validate from a SQL database. When you validate data using SQL, all compute is managed by your database, and only Validation Results are returned.
 
+.. content-tabs::
 
-.. admonition:: Prerequisites -- this how-to guide assumes you have already:
+    .. tab-container:: tab0
+        :title: Show Docs for V2 (Batch Kwargs) API
 
-  - :ref:`Set up a working deployment of Great Expectations <tutorials__getting_started>`
-  - :ref:`Configured a SQL datasource <how_to_guides__configuring_datasources>`
-  - Identified a ``table`` name, ``view`` name, or ``query`` that you would like to use as the data to validate.
-  - Optionally, configured a Batch Kwargs Generator to support inspecting your database and dynamically building Batch Kwargs.
+        .. admonition:: Prerequisites -- this how-to guide assumes you have already:
 
+          - :ref:`Set up a working deployment of Great Expectations <tutorials__getting_started>`
+          - :ref:`Configured a SQL Datasource <how_to_guides__configuring_datasources>`
+          - Identified a ``table`` name, ``view`` name, or ``query`` that you would like to use as the data to validate.
+          - Optionally, configured a Batch Kwargs Generator to support inspecting your database and dynamically building Batch Kwargs.
 
-Steps
------
 
-#. Construct Batch Kwargs that describe the data you plan to validate.
+        Steps
+        -----
 
-    For example, if your configured datasource is named "my_db" and you plan to validate data in a view called "my_view" located in the "my_schema" schema:
+        #. Construct Batch Kwargs that describe the data you plan to validate.
 
-    - To manually build ``table`` or ``view``-based Batch Kwargs, create a dictionary:
+            For example, if your configured datasource is named "my_db" and you plan to validate data in a view called "my_view" located in the "my_schema" schema:
 
-        .. code-block:: python
+            - To manually build ``table`` or ``view``-based Batch Kwargs, create a dictionary:
 
-            batch_kwargs = {
-                "datasource": "my_db",
-                "schema": "my_schema",  # schema is optional; default schema will be used if it is omitted
-                "table": "my_view"  # note that the "table" key is used even to validate a view
-            }
+                .. code-block:: python
 
-    - To use a Batch Kwargs Generator, call the ``build_batch_kwargs`` method:
+                    batch_kwargs = {
+                        "datasource": "my_db",
+                        "schema": "my_schema",  # schema is optional; default schema will be used if it is omitted
+                        "table": "my_view"  # note that the "table" key is used even to validate a view
```

Comment:
Would be nice if you could briefly explain why the table key could be used to do this - not entirely intuitive!